CONTRIBUTING.md

```diff
@@ -75,7 +75,7 @@ $ git rebase -i HEAD~3
 If your branch conflicts with the master branch, you will need to rebase and repush it with the following commands:
 
 ``` bash
-$ git remote add upstream git@github.com:php-http/repo-name.git
+$ git remote add upstream git@github.com:orga/repo-name.git
```

Comment:
but then we could also have a file with just a link to a wiki page in
httplug, or a documentation page on httplug.io. then we would not need
to keep the file in sync.

on the other hand, that file is not likely to change often. i don't mind
either way.